pkg/sql/schemachanger/scbuild/builder_state.go

```diff
@@ -173,6 +175,18 @@ func (b *builderState) CurrentUserHasAdminOrIsMemberOf(role username.SQLUsername
 
 var _ scbuildstmt.TableHelpers = (*builderState)(nil)
 
+// NextZoneConfigID implements the scbuildstmt.TableHelpers interface.
+func (b *builderState) NextZoneConfigID(table *scpb.Table) uint32 {
+	tblElts := b.QueryByID(table.TableID)
+	zoneConfigID := uint32(0)
+	scpb.ForEachTableZoneConfig(tblElts, func(current scpb.Status, target scpb.TargetStatus, e *scpb.TableZoneConfig) {
```

Comment:
sorry... I couldn't find where `ForEachTableZoneConfig` function is defined either in master and in this pr... Would a multi-region table have more than one zone config? Or `ForEachTableZoneConfig` function is actually also looping through all the sub-zone configs as well?